packages/libs/e2e-test-utils/src/lib/env.ts

```diff
@@ -25,12 +25,18 @@ const baseEnvSchema = {
   TEST_FUNDER_PRIVATE_KEY: z.string(),
   TEST_APP_MANAGER_PRIVATE_KEY: z.string(),
   TEST_APP_DELEGATEE_PRIVATE_KEY: z.string(),
-  TEST_PLATFORM_USER_WALLET_OWNER_PRIVATE_KEY: z.string(),
+  TEST_AGENT_WALLET_PKP_OWNER_PRIVATE_KEY: z.string(),
   YELLOWSTONE_RPC_URL: z.string().optional().default('https://yellowstone-rpc.litprotocol.com/'),
-  BASE_RPC_URL: z.string().optional().default('https://base.llamarpc.com'),
   // Smart Account (optional - only needed when enableSmartAccount=true)
   SMART_ACCOUNT_CHAIN_ID: z.string().optional(),
+  ALCHEMY_RPC_URL: z.string().optional(),
+  // ZeroDev
   ZERODEV_RPC_URL: z.string().optional(),
+  // Crossmint
+  CROSSMINT_API_KEY: z.string().optional(),
+  // Safe
+  SAFE_RPC_URL: z.string().optional(),
+  PIMLICO_RPC_URL: z.string().optional(),
```

Comment:
These should be defined in the ability e2e tests like [this](https://github.com/LIT-Protocol/Vincent/blob/e8a2218a91f449db3072100dc868ffe4a35e6111/packages/apps/ability-aerodrome-swap/test/e2e/swap-with-sponsorship.spec.ts#L37-L40) as they aren't relevant to other e2e tests that don't use smart account (although going forward everything is going to be using smart accounts, so maybe it makes sense to add it to the base ENV?)